7_grovers_search/qsharp/01_oracles/test_oracles.py

```diff
@@ -21,15 +21,15 @@ def test_mark_states(n, markedStates):
        f"GroversSearch.Test.FMarkStates(_, {markedStates}))")
 
 
-@pytest.mark.parametrize("n, marked_states", test_cases)
-def test_apply_phase_oracle(n, marked_states):
+@pytest.mark.parametrize("n, markedStates", test_cases)
```

Comment:
I'm following Python convention for variable naming here, for consistency with Qiskit tests and other Python code, so I think it makes sense to keep it as is. Can you please revert the change to this file? 

The rest of the PR looks good! I'm planning to go through to the code and text for all chapters once more doing this kind of cleanup in the next two months, now that I'm done with the first draft :-)

Reply:
I reverted the renaming as you suggested. Overall this is terrific work and I enjoyed delving through the code. Looking forward to seeing the finished book! 🙂